CodeGeneration/Package.swift

```diff
@@ -11,7 +11,7 @@ let package = Package(
     .executable(name: "generate-swiftsyntax", targets: ["generate-swiftsyntax"])
   ],
   dependencies: [
-    .package(url: "https://github.com/apple/swift-syntax.git", revision: "94b9021a2e461fc9a4b3bda6f4734119b52e8094"),
+    .package(url: "https://github.com/apple/swift-syntax.git", revision: "55349f8019fb129781af3f988011b20b66989664"),
```

Comment:
Also updated the swift-syntax version. Even there is noting new, just so we don't get too far behind.